mitiq/benchmarks/volume_circuits.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (C) 2022 Unitary Fund
```

Comment:
I suggest calling this file either `quantum_volume` or `quantum_volume_circuits` for homogeneity with the other modules.